src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -10,7 +10,24 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::ObjectManager
 {
+    struct object_repository_item
```

Comment:
this should be within the tight packing. I know its not going to change but best to be consistent with all loco_globals being tight packed.